active/39-group-chat/README.md

```diff
@@ -26,42 +26,35 @@ Users want to have a convenient and powerful group chat comparable to the market
 ### Minimum Viable Product
 
 - Goal Date: (TBD)
-- Description: The MVP provides users with basic moderation tools so that they can block users, filter posts with certain phrases, and reduce spam.
+- Description: Status provides a peer-to-peer chat plus one open chat room for the global exchange between the first users. This room is moderated to avoid misusage.
```

Comment:
Nice description! We can narrow this down a bit more though as there won't be chat moderation in the MVP of group chat.

This can just be:

```
Status provides peer-to-peer chat.
```